src/test/java/com/sendgrid/LicenseTest.java

```diff
@@ -0,0 +1,27 @@
+package com.sendgrid;
```

Comment:
I am thinking that this isn't running due to being in the wrong path? We have some other tests here: https://github.com/sendgrid/sendgrid-java/tree/master/src that are running in Travis. Can you please move this file there?